Windows/__init__.py

```diff
@@ -86,6 +86,9 @@ def from_param(self, obj):
         if isinstance(obj, str):
             if issubclass(self.pointer_type, (POINTER(Int16), POINTER(UInt16))):
                 return obj
+        elif isinstance(obj, int):
+            if issubclass(self.pointer_type, (POINTER(Int32), POINTER(UInt32))):
+                return byref(self.pointer_type._type_(obj))

```

Comment:
I suppose it is impossible to pass python's int by reference.  `obj` can not be changed by function.

Reply:
when we are passing python class as argument instead of ctypes classes, we don't care about the object reference.
This is creating reference to move the final value correctly to that function. (auto push the integer to type of argument then assign to its reference)
But if you have better idea, I'll be happy to know.